src/go/rpk/pkg/cli/cmd/status.go

```diff
@@ -178,19 +189,14 @@ func getMetrics(
 		}
 		return res, errors.Errorf("Error gathering metrics:%s", msgs)
 	}
+	log.Infof("metrics: %v", m)
```

Comment:
I think it would be nicer if logging was done in the calling function, it has access to the data.

Reply:
I missed this, this wasn't supposed to be checked in.